<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllergiesDialog</class>
 <widget class="QDialog" name="AllergiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AllergiesDialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="TextEdit" name="currentEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="TextEdit" name="pastEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>АЛЕРГИИ:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отказ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>МИНАЛИ ЗАБОЛЯВАНИЯ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>НАСТОЯЩИ ЗАБОЛЯВАНИЯ:</string>
   </property>
  </widget>
  <widget class="TextEdit" name="allergiesEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNoEditMenu</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>View/uiComponents/qt_derived/TextEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>allergiesEdit</tabstop>
  <tabstop>currentEdit</tabstop>
  <tabstop>pastEdit</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
